### PR TITLE
mobXDevTools Flag & Update normalize.css

### LINKED ISSUE #3: Broken Toggles?
First off, awesome work on this add-on! It's really useful so far!

screen shot 2016-11-28 at 10 39 53 am

One thing I did find was that there seems to be something off with the buttons in the bottom right that the add-on adds. Maybe i'm missing something, but what do those toggles do? Are they broken? They don't seem to do anything when I click them or toggle them?

### PR DESCRIPTION
**Pull request by bluetidepro**
- Add a `bool` to turn off/on the `mobXDevTools` (Closes #3)
- Updates outdated `normalize.css` from `v4.1.1` to `v5.0.0`
- Bumps version of `storybook-host` to `1.0.8`

### CODE REVIEW

**Comment by philcockfield** [comment]
Oh hey @bluetidepro - thanks for doing this.  I'll merge this in, but I think it probably makes sense to drop the dependency on MobX and make it generic for any kind of library dev tools like this (ie. The Redux version too).

File: README.md [hunk 1]
```diff
@@ -54,7 +55,8 @@ host({
   padding: <number | string>,
 });
 ```
-
+#### `mobXDevTools: boolean`
+Flag indicating if the mobXDevTools at the bottom should be shown.  Default: `true`.
```
Comment (philcockfield) [inline README.md]: Dude!  Documentation FTW 😄

**Comment by philcockfield** [comment]
Oh hey @bluetidepro - thanks for doing this.  I'll merge this in, but I think it probably makes sense to drop the dependency on MobX and make it generic for any kind of library dev tools like this (ie. The Redux version too).

**Comment by philcockfield** [comment]
https://github.com/philcockfield/storybook-host/issues/5

**Comment by philcockfield** [comment]
Published to NPM.

**Comment by bluetidepro** [comment]
@philcockfield Yeah, I agree that eventually it makes more sense to make that generic, but at least for now it's an optional thing! Thanks! 👍  :)

**Pull request merged**